swayfmt/src/items/item_fn/tests.rs

```diff
@@ -88,3 +88,35 @@ fmt_test_item!(  fn_nested_if_lets
     };
 }"
 );
+
+fmt_test_item!(  fn_conditional_with_comment
+"fn conditional_with_comment() {
+    if true {
+        // comment here
+    }
+}",
+intermediate_whitespace
+"fn conditional_with_comment() {
```

Comment:
Forgive my ignorance but how is this test supposed to work? Is the 2nd code block supposed to be identical to the first?

Reply:
The first is the "expected" and the second is the input. So in this case I'm asserting that the formatter doesn't change it (which is the bug)

Reply:
agh i see, thanks for clarifying.